src/apps/common/submodules/navigationWizard/navigationWizard.js

```diff
@@ -118,6 +119,12 @@ define(function(require) {
 				stepId: navigationWizardFlags.currentStep
 			});
 
+			if (navigationWizardFlags.askForConfirmationBeforeLeave) {
+				$(window).on('beforeunload', function() {
```

Comment:
When binding events on `window` or any top DOM element, you should always think about unbinding them whenever possible.

Reply:
I forgot about it, thanks :+1: